Site/webapp/wdkCustomization/js/client/components/questions/InternalGeneDataset.tsx

```diff
@@ -161,35 +161,19 @@ function InternalGeneDatasetContent(props: Props) {
     shouldChangeDocumentTitle
   );
 
-  const activeTab = useMemo(() => {
-    if (!questionNamesByDatasetAndCategory || !selectedDataSetRecord || !displayCategoriesByName) return;
-    let tab;
-    for (const property in questionNamesByDatasetAndCategory[selectedDataSetRecord['dataset_name']]) {
-      if (questionNamesByDatasetAndCategory[selectedDataSetRecord['dataset_name']][property] === searchName) {
-        tab = displayCategoriesByName[property].displayName
-      }
-    }
-    return tab;
-  }, [questionNamesByDatasetAndCategory, selectedDataSetRecord, displayCategoriesByName, searchName]);
-
   const changeTabHandler = useCallback(
-    (selectedTabDisplayName: string) => {
+    (selectedTabKey: string) => {
       if (
         !questionNamesByDatasetAndCategory || 
         !selectedDataSetRecord || 
         !displayCategoriesByName ||
```

Comment:
Now that we're out of the business of looking up display names by search name, I guess we don't need check for these three values anymore?